test/NVQPP/custom_pass.cpp

```diff
@@ -5,11 +5,12 @@
  * This source code and the accompanying materials are made available under    *
  * the terms of the Apache License 2.0 which accompanies this distribution.    *
  ******************************************************************************/
+
+// REQUIRES: c++20
```

Comment:
Here we need to make a judgement call regarding the requirements for defining an using a custom pass plugin.
Does the fact that we compile CUDA Quantum with C++20 mean anyone building a pass plugin also needs to compile with the same C++ standard? Our simulators (that are loaded dynamically), for example are compiled with C++17, so it is not clear to me why this should be a requirement. If it is a requirement, we need to document this. If not, this test needs to be fixed and reenabled.